modal.css

```diff
@@ -0,0 +1,251 @@
+.modal-popup {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  background-color: #191765;
+  background-image: url('./image/Background-image/top-circles.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+  box-sizing: border-box;
+  opacity: 0;
+  visibility: hidden;
+  transform: scale(1.1);
+  transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+  display: flex;
+  justify-content: center;
+}
+
+.show-modal {
+  opacity: 1;
+  visibility: visible;
+  transform: scale(1);
+  transition: visibility 0s linear 0s, opacity 0.25s 0s, transform 0.25s;
+}
+
+.popup-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 8px;
+  width: 86%;
+  margin: auto;
+  background-color: rgba(255, 255, 255, 0.16);
+  border: 1px solid #fff;
+  box-shadow: 0 48px 48px rgba(37, 47 137, 0.08);
+}
+
+.ar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  width: 85%;
+  gap: 8px;
+  height: fit-content;
+}
+
+.title {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 28px;
+  padding-left: 2%;
+  margin-top: 2%;
+}
+
+.close-popup-img {
+  color: #fff;
+  display: block;
+  width: 8%;
+  float: right;
+  padding-top: 5px;
+  padding-right: 3%;
+}
```

Comment:
- [ **OPTIONAL** ] **Important:** I think it would be better if you add a` cursor pointer` property on your close popup button to help users know that it is clickable and if possible use a `<a>` or `<button>` tag on it in HTML file for best practices.